src/plugins/plus-menu/PlusMenu.tsx

```diff
@@ -11,12 +11,14 @@ import {
   faCode,
   faExternalLinkSquareAlt,
   faImage,
+  faSort,
 } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
   onClickCodeButton,
   onClickIframeButton,
   onClickImageButton,
+  onClickSortButton,
```

Comment:
I'd try to be consistent with naimg things. As the plugin is now called `sorting-task`, I think we should name that function `onClickSortingTaskButton`.

```suggestion
  onClickSortingTaskButton,
```